### PR TITLE
Add PromptSpriteFormatter to settings to allow for universal formatting from PromptText

### PR DESCRIPTION
**Pull request by thsbrown**
See https://github.com/simonoliver/InputSystemActionPrompts/issues/2

### CODE REVIEW

**Comment by simonoliver** [comment]
Hey, thanks for this, feels like a neat solution. I'll try and review this week. Couple initial thoughts;

* Having {0} as the sprite substitution pattern in PromptSpriteFormatter feels a little unintuitive. Maybe this could be {SPRITE}?
* Could you add instructions for this to the README file? Just a quick example like you have in the comments explaining what you can use this for and how to do it.

Thanks again for the contribution!
Simon

**Comment by thsbrown** [comment]
Thanks for the feedback!

I will definitely add to the README. I also thought the `{0}` was a little unintuitive. Additionally, I definitely think `{SPRITE}` is a step in the right direction and will make that change (there might be an event more intuitive way to do this, but progress over perfection right 🙂).

**Comment by thsbrown** [comment]
Ok, just made the updates. Let me know if the README documentation works. Feel free to update it to make it more clear 👍 .

File: Runtime/InputSystemActionPrompts/InputDevicePromptSystem.cs [hunk 1]
```diff
@@ -128,6 +128,13 @@ public static string InsertPromptSprites(string inputText)
             foreach (var tag in foundTags)
             {
                 var replacementTagText = GetActionPathBindingTextSpriteTags(tag);
+                
+                //if PromptSpriteFormatter is empty for some reason return the text as if formatter was {SPRITE} (normally)
+                var promptSpriteFormatter = s_Settings.PromptSpriteFormatter == "" ? "{SPRITE}" : s_Settings.PromptSpriteFormatter;
```
Comment (simonoliver) [inline Runtime/InputSystemActionPrompts/InputDevicePromptSystem.cs:133]: Actually,  sorry a diff suggestion: 
1. Move "{SPRITE}" to be a public const string in InputDevicePromptSystem and use in all places
2. Check that the s_Settings.PromptSpriteFormatter contains {SPRITE} on Initialise. If it doesn't, log an error to the effect that it's missing.

**Review by simonoliver — COMMENTED** [review]
Hey, all looks great, thanks for this. Have added one suggestion

File: Runtime/InputSystemActionPrompts/InputDevicePromptSystem.cs (hunk 1 above)
Comment (thsbrown) [reply Runtime/InputSystemActionPrompts/InputDevicePromptSystem.cs:133]: Hey man you're totally right on the suggestions! Sorry been a bit busy but will try to get to this asap so we can get this merged in asap!

**Comment by thsbrown** [comment]
So sorry this took forever man! All changes should be implemented as requested. Let me know if there's any further issues 👍.

**Comment by thsbrown** [comment]
After reviewing the code once again, I thought that `PromptSpriteFormatterSpritePlaceholder` made a bit more sense in the PromptSettings rather than PromptSystem.

Let me know if that works.

**Review by simonoliver — APPROVED** [review]
Thanks - this looks great!

**Pull request merged**